x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/ExceptionStacktrace.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiTitle, EuiSpacer } from '@elastic/eui';
+import { idx } from '@kbn/elastic-idx/target';
+import { Exception } from '../../../../../typings/es_schemas/raw/ErrorRaw';
+import { Stacktrace } from '../../../shared/Stacktrace';
+import { CauseStacktrace } from '../../../shared/Stacktrace/CauseStacktrace';
+
+interface ExceptionStacktraceProps {
+  codeLanguage?: string;
+  exceptions: Exception[];
+}
+
+export function ExceptionStacktrace({
+  codeLanguage,
+  exceptions
+}: ExceptionStacktraceProps) {
+  const title = idx(exceptions, _ => _[0].message);
+
+  return (
+    <>
+      <EuiTitle size="xs">
+        <h4>{title}</h4>
+      </EuiTitle>
+      <EuiSpacer size="l" />
```

Comment:
I think it looks better without this spacer.

<img width="960" alt="Screenshot 2019-10-29 at 10 22 19" src="https://user-images.githubusercontent.com/4104278/67754161-40920480-fa36-11e9-9c4d-3ffcc35e1e80.png">